watcher/local/watcher_test.go

```diff
@@ -92,3 +197,109 @@ func newWatcher(t *testing.T, rs channel.RegisterSubscriber) *local.Watcher {
 func makeSignedStateWDummySigs(params *channel.Params, state *channel.State) channel.SignedState {
 	return channel.SignedState{Params: params, State: state}
 }
+
+// randomTxsForSingleCh returns "n" transactions for a random channel.
+func randomTxsForSingleCh(rng *rand.Rand, n int) (*channel.Params, []channel.Transaction) {
+	params, initialState := channeltest.NewRandomParamsAndState(rng, channeltest.WithVersion(0),
+		channeltest.WithNumParts(2), channeltest.WithNumAssets(1), channeltest.WithIsFinal(false), channeltest.WithNumLocked(0))
+
+	txs := make([]channel.Transaction, n)
+	for i := range txs {
+		txs[i] = channel.Transaction{State: initialState.Clone()}
+		txs[i].State.Version = uint64(i)
+	}
+	return params, txs
+}
+
+// trigger is used for triggering events on the mock adjudicator subscription.
+//
+// Adjudicator subscription blocks until it is closed.
+type trigger struct {
+	handle    chan chan time.Time
```

Comment:
Could you please make this a typedef? As i understand, the mockify needs this type of channel, but only checks if it is closed or receives something.  
Maybe there is also a function which takes `chan struct{}`.

Reply:
Yes, your understanding is correct. 

I couldn't find a function that takes`chan struct {}`. Looking at [their documentation](https://pkg.go.dev/github.com/stretchr/testify/mock#Call.WaitUntil), I think, they chose `chan time.Time` because
1. It can be used as signal by closing (which is what we do).
2. It can be used with a time.Timer.

Reply:
Since, it is only in once place, I think comments would be better than a type def. 

See be938d5. Does this work ? Or should I add a type def ?